addons/interaction/CfgVehicles.hpp

```diff
@@ -251,6 +251,15 @@ class CfgVehicles {
                 statement = QUOTE([ARR_3(_player,_target,1)] call DFUNC(tapShoulder));
                 exceptions[] = {"isNotSwimming"};
             };
+            class ACE_OpenBackpack {
+                displayName = "$STR_ACTION_OPEN_BAG";
+                position = QUOTE(call DFUNC(getBackpackPos));
+                distance = 3.0;
+                condition = QUOTE([ARR_2(_target,_player)] call DFUNC(canOpenBackpack));
```

Comment:
```suggestion
                condition = QUOTE(call DFUNC(canOpenBackpack));
```

Reply:
I just left it as that to match the other actions